### DISCUSSION TITLE
Is it possible to hide the taskbar icon for SplashWindow?

### DISCUSSION THREAD

**Discussion by aj3423** [fyne-io/fyne#3344] [Q&A] [answered]
My app is used for voice speech recognition, I want to show the speech text directly on the screen.

I found a transparent `SplashWindow` works, but it also shows an item on the task bar, is there any way to hide it?

![2022-10-20_00-04](https://user-images.githubusercontent.com/4710875/196744893-bd4bd42f-3a69-47a4-a6e7-6ba60bbe48db.png)

My code:
```golang
package main

import (
	"image/color"

	"fyne.io/fyne/v2"
	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/canvas"
	"fyne.io/fyne/v2/driver/desktop"
	"fyne.io/fyne/v2/theme"
	"github.com/go-gl/glfw/v3.3/glfw"
)

type myTheme struct{}

// Set canvas bg color to transparent
func (m myTheme) Color(name fyne.ThemeColorName, variant fyne.ThemeVariant) color.Color {
	transparent := color.NRGBA{R: 0, G: 0, B: 0, A: 0}
	return transparent
}

// Just use default font
func (m myTheme) Icon(name fyne.ThemeIconName) fyne.Resource {
	return theme.DefaultTheme().Icon(name)
}

// Just use default font
func (m myTheme) Font(style fyne.TextStyle) fyne.Resource {
	return theme.DefaultTheme().Font(style)
}

func (m myTheme) Size(name fyne.ThemeSizeName) float32 {
	return theme.DefaultTheme().Size(name)
}

var blue = color.NRGBA{R: 0x0d, G: 0x93, B: 0xb4, A: 224}

func newText(text string) *canvas.Text {
	t := canvas.NewText(text, blue)
	t.TextSize = 40
	return t
}

func main() {
	a := app.New()

	// these two lines are for transparent splash window
	a.Settings().SetTheme(&myTheme{})
	if glfw.Init() == nil {
		glfw.WindowHint(glfw.TransparentFramebuffer, glfw.True)
	}

	if drv, ok := a.Driver().(desktop.Driver); ok {
		w := drv.CreateSplashWindow()
		w.SetContent(newText("aaaaaa"))
		w.Show()
	}

	a.Run()
}
```

**Comment by Jacalz** [comment, marked as answer]
I believe that https://github.com/fyne-io/fyne/issues/3156 is the issue related to what you want to do. I don't think there is much to do until that is implemented, unfortunately. Similar to https://github.com/fyne-io/fyne/discussions/3237 as well.

**Reply by andydotxyz** [reply]
I don't think we should remove window interactions like that to not appear for visible windows.
A textual overlay as described is a very OS specific extension and I'm not totally sure it belongs in a toolkit that is focused on platform agnostic...?

**Reply by Jacalz** [reply]
Yeah. I think I agree. Seems like it could be problematic.
